### ISSUE TITLE
[10.0e] UnicodeEncodeError when trying to drill down to General Ledger from Trial Balance Report

### ISSUE THREAD

**Issue by yostashiro** [odoo/odoo#19853]
### Impacted versions:

10.0 Enterprise (haven't checked other versions)

### Steps to reproduce:

- Install Japanese language and change your language to Japanese.
- Go to *Accounting > Reports > Legal Statements >Trial Balance*, and try to drill down to General Ledger from an account.

### Current behavior:

```
Odoo Server Error
Traceback (most recent call last):
  File "/home/ytashiro/ws/10.0/odoo/odoo/http.py", line 640, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/ytashiro/ws/10.0/odoo/odoo/http.py", line 677, in dispatch
    result = self._call_function(**self.params)
  File "/home/ytashiro/ws/10.0/odoo/odoo/http.py", line 333, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/ytashiro/ws/10.0/odoo/odoo/service/model.py", line 101, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/ytashiro/ws/10.0/odoo/odoo/http.py", line 326, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/ytashiro/ws/10.0/odoo/odoo/http.py", line 935, in __call__
    return self.method(*args, **kw)
  File "/home/ytashiro/ws/10.0/odoo/odoo/http.py", line 506, in response_wrap
    response = f(*args, **kw)
  File "/home/ytashiro/ws/10.0/odoo/addons/web/controllers/main.py", line 885, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/home/ytashiro/ws/10.0/odoo/addons/web/controllers/main.py", line 877, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/ytashiro/ws/10.0/odoo/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/home/ytashiro/ws/10.0/odoo/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/ytashiro/ws/10.0/enterprise/account_reports/models/account_report_context_common.py", line 482, in get_html_and_data
    lines = self.get_report_obj().get_lines(self)
  File "/home/ytashiro/ws/10.0/enterprise/account_reports/models/account_general_ledger.py", line 40, in get_lines
    return self.with_context(new_context)._lines(line_id)
  File "/home/ytashiro/ws/10.0/enterprise/account_reports/models/account_general_ledger.py", line 332, in _lines
    'columns': [datetime.strptime(line.date, DEFAULT_SERVER_DATE_FORMAT).strftime(date_format), name, partner_name, currency,
UnicodeEncodeError: 'ascii' codec can't encode character u'\u5e74' in position 2: ordinal not in range(128)
```

### Expected behavior:

No such error because of the date format.

The issue comes from the date format of Japanese `%Y年%m月%d日`.  When you change it to `%Y/%m/%d` for example, there is no error.
![image](https://user-images.githubusercontent.com/7766939/31116591-3e6f30b4-a859-11e7-9798-edafb19a5c34.png)

### Video/Screenshot link (optional):

![2017-10-03_16-38-19](https://user-images.githubusercontent.com/7766939/31116827-0aee1e02-a85a-11e7-80e8-9523de849adb.gif)

**Comment by Yenthe666** [comment]
@mart-e this is languages related. Could you have a look when you've got some free time?

**Comment by mart-e** [comment]
Thanks, should be fixed at 4f0ebfc54
